fe/fe-core/src/main/java/com/starrocks/sql/optimizer/MvRewriteContext.java

```diff
@@ -93,21 +88,6 @@ public ScalarOperator getMvPruneConjunct() {
         return mvPruneConjunct;
     }
 
-    public List<ScalarOperator> getQueryJoinOnPredicates() {
```

Comment:
why remove this?

Reply:
Since join's on predicates have been checked in `computeCompatibility`, no need to check this any more?
```
            if (! ScalarOperator.isEquivalent(queryJoin.getOnPredicate(), (mvJoin.getOnPredicate()))) {
                logMVRewrite(mvRewriteContext, "join predicate is different {} != {}", queryJoin.getOnPredicate(),
                        mvJoin.getOnPredicate());
                return false;
            }
```

However maybe `ScalarOperator.isEquivalent` cannot handle some `Equivalent` operator, maybe we can optimize it later?

eg:
Q  : a left b on a.c1=b.c1 and a.c2 > 10
MV : a left b on a.c1=b.c1 and a.c2 >= 9